docs/Filesystem-Paths.md

```diff
@@ -25,9 +25,9 @@ only modified when a newer version of the app is being installed.
 Since Brim is most often launched directly from the GUI environment in your OS,
 you'll typically not need to access the application binaries directly. One
 exception is if you wish to make use of the command line binaries that are
-bundled with the app, such as [`zq`](https://github.com/brimdata/zed/tree/main/cmd/zq),
-[`zed`](https://github.com/brimdata/zed#quick-start),
-[`brimcap`](https://github.com/brimdata/brimcap), and so forth. These can be
+bundled with the app, such as [`zq`](https://zed.brimdata.io/docs/commands/zq/),
+[`zed`](https://zed.brimdata.io/docs/commands/zed/),
+[`brimcap`](https://github.com/brimdata/brimcap/blob/main/README.md), and so forth. These can be
```

Comment:
@nwt: Ok, personal preference I guess. I suggested linking to the Brimcap README because the links to `zq` and `zed` were pointing to "docs that describe how to use them" rather than "their repos", and in this regard I find a direct link to the Brimcap README a bit more apples-to-apples. But it ultimately lands at a page with the same content, so I'm happy to go with your changes.